### PR TITLE
Nazgul bio

### PR DESCRIPTION
**Pull request by NazgulM**
First, when I create branch used npm i, for installation packages; Thanks Tamer for help.
After 
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

**Comment by talmurshidi** [comment]
Very good @NazgulM 
I noticed that you have error with linting `Code style issues found in the above file(s). Forgot to run Prettier?`
You can use `npm run format` before pushing your changes and this should fix the problem.

**Comment by talmurshidi** [comment]
Hello @NazgulM 
As @danielhalasz  said: the images must be inside `assets` folder so please move the image inside that folder in student-bios folder.

**Comment by danielhalasz** [comment]
@NazgulM , please do a `git pull origin main` to make sure you are on the latest version of the repo..and to avoid the file conflict

**Comment by NazgulM** [comment]
I guess now problem is solved

**Comment by talmurshidi** [comment]
@NazgulM , I noticed the branched is not up-to-date so please:

1. Make sure the branch is up-to-date using command `git pull upstream main` .
2. Check the work again and push it to the same branch and PR.

We will check it again after pushing the changes and update.

File: student-bios/NazgulM.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# About me
+
+## Nazgul Maniiasova
+
+<img src="Naza.jpg" width="150" height="100">
```
Comment (talmurshidi) [inline student-bios/NazgulM.md]: This is a HTML tag and it is not working with Markdown.
To fix it:
`My photo`

**Review by talmurshidi — APPROVED** [review]
Very good job, well done.

**Pull request merged**